common/http/src/main/java/io/helidon/common/http/HashParameters.java

```diff
@@ -110,7 +137,7 @@ public static HashParameters concat(Parameters... parameters) {
     /**
      * Creates new instance of {@link HashParameters} as a concatenation of provided parameters.
      *
-     * @param parameters parameters to concat.
+     * @param parameters parameters to concatenate.
      * @return a concatenation, never {@code null}.
      */
     public static HashParameters concat(Iterable<Parameters> parameters) {
```

Comment:
As this is now a public class, an explanation of what `concatenation` means in this context would be good. I have no idea.

Reply:
Ok will do.